server/auth/store.go

```diff
@@ -493,7 +493,7 @@ func (as *authStore) UserChangePassword(r *pb.AuthUserChangePasswordRequest) (*p
 	var password []byte
 	var err error
 
-	if !user.Options.NoPassword {
+	if user.Options == nil || !user.Options.NoPassword {
```

Comment:
Nit: It's great if you can add a comment which describes the reason of backward compatibility.

Reply:
done